examples/14_react-tweet/vite.config.ts

```diff
@@ -1,15 +1,18 @@
 /** @type {import('vite').UserConfig} */
-export default ({ mode }: { mode: string }) => ({
-  ...(mode === 'development' && {
-    optimizeDeps: {
-      include: [
-        'react-dom/client',
-        'react-tweet > use-sync-external-store/shim/index.js',
-        'react-tweet > date-fns/format/index.js',
-      ],
-    },
-    ssr: {
-      noExternal: ['react-tweet'],
-    },
-  }),
-});
+export default ({ mode }: { mode: string }) => {
+  if (mode === 'development') {
+    return {
+      optimizeDeps: {
+        include: [
+          'react-dom/client',
```

Comment:
btw, I added this because I saw some warnings.
But, because this should be added by react plugin already, should we remove it?

Reply:
Yea, removing it also worked. done.